.circleci/config.pre.yml

```diff
@@ -168,6 +168,23 @@ jobs:
               }
             }]
           }
+  "docs-check-checksums":
```

Comment:
No, this should be part of the generator itself. It should just error if a checksum doesn't match

Reply:
Okay. The newest commits get rid of the continuous integration test.

Reply:
Okay. The new commits don't have a continuous integration test.